README.md

```diff
@@ -43,21 +43,21 @@ npm install
 
 ## New Token Methods
 
-Líf token is ERC20 compatible but it also has two more methods to allow the transference of data between users/contracts.
+Líf token is ERC20 compatible but it also has two more methods to allow the transference of data and execution of calls between users/contracts.
 
 ### transferData
 
 Transfer tokens from one address to another.
 ```
-transferData(address _to, uint _value, string _data) => (bool success)
+transferData(address to, uint value, bytes data, bool doCall) => (bool success)
```

Comment:
What's the semantic difference of both values of `doCall`?

I mean, when it's false it does it mean that data is just data, but when it's true it means that data has all the information to execute a method in the contract (I guess it has the method name and the args)? If so, I think we could make the api more clear by separating this into 2 separate methods, one to transfer data and another for method calls. Or at least explain a bit more about `doCall`